mesatee_services/kms/proto/build.rs

```diff
@@ -15,33 +15,34 @@
 // specific language governing permissions and limitations
 // under the License.
 
-#[cfg(not(feature = "mesalock_sgx"))]
-include!("../../common/prost_build_generator.rs");
+use std::env;
+use std::process::Command;
+use std::str;
 
-#[cfg(not(feature = "mesalock_sgx"))]
 fn main() {
+    let out_dir = env::var("OUT_DIR").expect("$OUT_DIR not set. Please build with cargo");
     println!("cargo:rerun-if-changed=src/kms.proto");
-    let src = PathBuf::from("src");
-    let output = src.join("prost_generated");
-    if !output.exists() {
-        std::fs::create_dir(&output).expect("failed to create prost_generated dir");
+    println!("cargo:rerun-if-changed=build.rs");
+    let c = Command::new("cargo")
```

Comment:
Will the cargo use vendored  libraries?

Reply:
No, an issue has been submitted: https://github.com/apache/incubator-teaclave/issues/181